Constructions/Induction_Schema_Example.ml

```diff
@@ -1,16 +1,19 @@
+needs "Constructions/epsilon.ml";;
+needs "Constructions/QuotationTactics.ml";;
 (*Testing a few proofs to make sure this definition works*)
 
 (*Trivial proof that x is effecitve in x + 3*)
-prove(effectiveIn `x:num` `x + 3`,
+prove(mk_neg (mk_not_effective_in `x:num` `x + 3` `y:num`),
```

Comment:
Why the change from a positive statement to a double-negative?

Reply:
Yes I guess that does look off- it's because I changed the default from effective-in to not-effective-in. I should still have an effective-in statement though, will add.